app/assets/javascripts/hyrax/channels/notifications.js

```diff
@@ -0,0 +1,27 @@
+App.cable.subscriptions.create("Hyrax::NotificationsChannel", {
+    connected: function(data) {
+        this.perform("update_locale", { locale: $('html').attr('lang') })
+    },
+
+    received: function(data) {
+        let count = data["notify_number"];
```

Comment:
Whenever a notification is created for a user, two bits of data are passed via the websocket: the number of unread notifications and a message like "You have 12 notifications" that is translated into the user's preferred locale (or the default locale if no preference exists).